### ISSUE TITLE
it is better to keep expiry data from oauth2session

### ISSUE THREAD

**Issue by andytzeng** [GoogleCloudPlatform/google-auth-library-python-oauthlib#18]
the helper function `credentials_from_session` helps to generate `Credentials` object from session. But, it does not carry **expires_at** data from `oauth2session.token`

It is better that the expiry can be carried to Credentials object. So, caller can check `expired` property of Credentials to know whether the token expired.

Thanks.

**Comment by jonparrott** [comment]
@andytzeng good point! Would you be willing to send a PR to do this? It seems like an easy fix.

**Comment by andytzeng** [comment]
sure, I can do that
